internal/controller/component_dependency_update_controller_test.go

```diff
@@ -316,6 +316,19 @@ var _ = Describe("Component nudge controller", func() {
 			Expect(len(renovatePipelines)).Should(Equal(1))
 			renovateCommand := strings.Join(renovatePipelines[0].Spec.PipelineSpec.Tasks[0].TaskSpec.Steps[0].Command, ";")
 			Expect(strings.Contains(renovateCommand, `'username':'image_repo_username'`)).Should(BeTrue())
+
+			Expect(renovatePipelines[0].Labels).Should(Equal(map[string]string{
+				"build.appstudio.redhat.com/type": "nudge",
+			}))
+
+			Expect(renovatePipelines[0].Annotations).Should(Equal(map[string]string{
+				"build.appstudio.redhat.com/nudged-components": "operator1-nudges operator2-nudges operator1-nudges operator2-nudges",
```

Comment:
Why do we have duplicated values?

Reply:
Sorry, I somehow missed the thread. I remembered asking you, but failed find the thread, so asked again...